src/main/java/org/broadinstitute/consent/http/service/DarCollectionService.java

```diff
@@ -811,19 +809,19 @@ private List<User> getDistinctAdminAndChairUsersForDAR(DataAccessRequest dar) {
   }
 
   private List<User> getDistinctAdminAndChairUsersForDatasetIds(List<Integer> datasetIds) {
-    List<User> admins = userDAO.describeUsersByRoleAndEmailPreference(UserRoles.ADMIN.getRoleName(),
-        true);
+    List<User> admins = userDAO.findUsersByRoleId(UserRoles.ADMIN.getRoleId());
     Set<User> chairPersons = userDAO.findUsersForDatasetsByRole(datasetIds,
-        Collections.singletonList(UserRoles.CHAIRPERSON.getRoleName()));
+        Collections.singletonList(UserRoles.CHAIRPERSON.getRoleId()));
     // Ensure that admins/chairs are not double emailed
-    // and filter users that don't want to receive email
     return Streams.concat(admins.stream(), chairPersons.stream())
-        .filter(u -> Boolean.TRUE.equals(u.getEmailPreference()))
```

Comment:
We actually do not want to filter users on this value. All email is funneled through one method, `EmailService.sendMessage` where we check for the user's email preference (in `SendGridAPI.sendMessage`). We record the state of the sent (or would have been sent) email at this point. If we pre-filtered here, we miss the chance to record what would have been sent.